src/Database/Query/Processors/RedshiftProcessor.php

```diff
@@ -2,8 +2,30 @@
 namespace YuK1\LaravelRedshift\Database\Query\Processors;
 
 use Illuminate\Database\Query\Processors\PostgresProcessor;
+use Illuminate\Database\Query\Builder;
 
 class RedshiftProcessor extends PostgresProcessor
 {
+    /**
+     * Process an "insert get ID" query.
+     *
+     * @param  \Illuminate\Database\Query\Builder  $query
+     * @param  string  $sql
+     * @param  array  $values
+     * @param  string|null  $sequence
+     * @return int
+     */
+    public function processInsertGetId(Builder $query, $sql, $values, $sequence = null)
+    {
+        $connection = $query->getConnection();
 
-}
\ No newline at end of file
+        $connection->insert($sql, $values);
+
+        $idColumn = $sequence ?: 'id';
```

Comment:
I am missing the part where and how the `$sequence` is coming in. Where do you supply the `$sequence`, that eventually is going to be passed into this method?

Reply:
In WBE code/app/Utils/Schema/DataCollector.php:384 ;) 
![Screenshot 2023-07-14 at 12 28 26](https://github.com/appsfortableau/laravel-redshift/assets/6881597/2ab1174d-e65b-48eb-ac9b-e0568830c84c)